web/components/input/text/TextInput/TextInput.js

```diff
@@ -50,7 +50,7 @@ const TextInput = ({
 TextInput.defaultProps = {
   className: '',
   sideLinkOnClick: () => {},
-  maxCharNumber: 100,
+  maxCharNumber: 256,
```

Comment:
My bad this should be 254 because we don't need the angle brackets https://stackoverflow.com/a/574698